webpack.config.js

```diff
@@ -115,6 +120,10 @@ module.exports = (env = {}) => {
           test: /\.handlebars?$/,
           use: ['handlebars-template-loader'],
         },
+        {
+          test: /\.styl(us)?$/,
+          use: [hot ? `style-loader` : MiniCssExtractPlugin.loader, `css-loader`, 'stylus-loader'],
```

Comment:
Hm. What part of the stack is loading stylus now?

Reply:
Vuetify uses stylus, so if we want to import any of their src files, this is needed.